thumbor/config.py

```diff
@@ -131,6 +131,7 @@
 Config.define('ALLOW_OLD_URLS', True, 'Indicates if encrypted (old style) URLs should be allowed', 'Security')
 Config.define('ENABLE_ETAGS', True, 'Enables automatically generated etags', 'HTTP')
 Config.define('MAX_ID_LENGTH', 32, 'Set maximum id length for images when stored', 'Storage')
+Config.define('GC_INTERVAL', 60, 'Set garbage collection interval in seconds', 'Performance')
```

Comment:
Sorry for not being clear enough. I see it as a _potential_ breaking change in a way that it may lead to a CPU usage rise in some scenarios we don't know about. We can't guarantee it won't. As we all know, forcing Python to do what it eventually is going to (collect collectable items) is not the real solution to the problem.

With that said, I believe it's okay supporting scheduled GC for those who can use and benefit from it, as long as it's an **_opt-in_**.

Having it on by default would lead us (Globo.com), for example, to either disable it or run massive tests to assert it doesn't affect our setup – remember we suffer from CPU usage and response time increase rather than OOM.